game_patch/input/mouse.cpp

```diff
@@ -111,6 +112,70 @@ ConsoleCommand2 ms_cmd{
     "ms <value>",
 };
 
+float scope_sensitivity_factor = 1.0f;
+float scanner_sensitivity_factor = 1.0f;
+
+void patch_scope_and_scanner_sensitivity()
+{
+    AsmWriter scope_writer(0x004309B1);
+    scope_writer.fmul<float>(AsmRegMem(&scope_sensitivity_factor));
+
+    AsmWriter scanner_writer(0x004309DE);
+    scanner_writer.fmul<float>(AsmRegMem(&scanner_sensitivity_factor));
```

Comment:
```suggestion
    AsmWriter{0x004309B1}.fmul<float>(AsmRegMem(&scope_sensitivity_factor));
    AsmWriter{0x004309DE}.fmul<float>(AsmRegMem(&scanner_sensitivity_factor));
```
Seems clearer.

FYI: Please see [C++ Core Guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es23-prefer-the--initializer-syntax) `ES.23: Prefer the {}-initializer syntax`. It is authorative.

Reply:
My bad on this one, thanks for the correction.